### PR TITLE
Update PL's to use CREATE/DROP EXTENSION

### PR DESCRIPTION
**Pull request by dyozie**
Move to documenting CREATE/DROP EXTENSION for pl/r, pl/perl, pl/python. Also update deprecation notices.

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/extensions/pl_perl.xml [hunk 1]
```diff
@@ -105,38 +105,25 @@
...
-Type "help" for help.
-
-testdb=# DROP LANGUAGE plperl;
-</codeblock>
-                <p> The default <codeph>DROP LANGUAGE</codeph> behavior is
-                    <codeph>RESTRICT</codeph>; the command fails if any existing objects (such as
-                    functions) depend on the language. Specify the <codeph>CASCADE</codeph> clause
-                    to also drop all dependent objects, including functions that you created with
-                    PL/Perl. </p>
+                            href="../sql_commands/CREATE_EXTENSION.xml">CREATE
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/extensions/pl_perl.xml]: link should be to DROP_EXTENSION

File: gpdb-doc/dita/ref_guide/extensions/pl_r.xml [hunk 2]
```diff
@@ -85,6 +85,18 @@
         </body>
       </topic>
     </topic>
+    <topic id="topic5" xml:lang="en">
+      <title>Enabling PL/R Language Support</title>
+      <body>
+        <p>For each database that requires its use, register the PL/R language with the SQL command
+            <codeph>CREATE EXTENSION</codeph>. Because PL/R is an untrusted language, only
+          superusers can register PL/R with a database. For example, run this command as the
+            <codeph>gpadmin</codeph> system to register the language with the database named
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/extensions/pl_r.xml]: system ^user^

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_LANGUAGE.xml [hunk 3]
```diff
@@ -169,8 +169,7 @@ CREATE [ OR REPLACE ] [ TRUSTED ] [ PROCEDURAL ] LANGUAGE <varname>name</varname
         <section id="section6">
             <title>Examples</title>
             <p>The preferred way of creating any of the standard procedural languages:</p>
-            <codeblock>CREATE LANGUAGE plperl;
-CREATE LANGUAGE plr;</codeblock>
+            <codeblock>CREATE LANGUAGE plperl;</codeblock>
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_LANGUAGE.xml]: Should this be `CREATE EXTENSION plperl;`?

File: gpdb-doc/dita/ref_guide/extensions/pl_r.xml [hunk 4]
```diff
@@ -85,6 +85,18 @@
...
+            <i>testdb</i>:</p>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/extensions/pl_r.xml]: maybe codeph testdb to be consistent with the other pl/xx instructions?

File: gpdb-doc/dita/ref_guide/extensions/pl_r.xml [hunk 5]
```diff
@@ -103,12 +115,14 @@
...
           <p>For a database that no longer requires the PL/R language, remove support for PL/R with
-            the SQL command <codeph>DROP LANGUAGE</codeph> or the Greenplum Database
-              <codeph>droplang</codeph> utility. Because PL/R is an untrusted language, only
-            superusers can remove support for the PL/R language from a database. For example,
-            running this command as the gpadmin user removes support for PL/R from the database
-            named <i>testdb</i>:</p>
-          <codeblock>$ droplang plr -d testdb</codeblock>
+            the SQL command <codeph>DROP EXTENSION</codeph>. Because PL/R is an untrusted language,
+            only superusers can remove support for the PL/R language from a database. For example,
+            run this command as the <codeph>gpadmin</codeph> user to remove support for PL/R from
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/extensions/pl_r.xml]: other language instructions reference "**system** user".  do we want to be consistent here?  (note that i am not sure why some of the pl/xxx's reference just "user" vs. "system user".  it is an os-level command and the instructions identify gpadmin, regardless of trusted vs. untrusted, wouldn't "user" implicitly imply the "system user"?)

File: gpdb-doc/dita/ref_guide/extensions/pl_r.xml [hunk 6]
```diff
@@ -103,12 +115,14 @@
...
+            the database named <i>testdb</i>:</p>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/extensions/pl_r.xml]: codeph testdb?

File: gpdb-doc/dita/ref_guide/extensions/pl_r.xml (hunk 5 above)
Comment (dyozie) [reply gpdb-doc/dita/ref_guide/extensions/pl_r.xml]: I agree that adding "system" here seems redundant.  Removed elsewhere.

**Pull request merged**